ote/src/cljs/ote/ui/validation.cljs

```diff
@@ -95,6 +95,12 @@
                    (year-month-and-day comparison-date)))
     (or message (str "Date must " (fmt/pvm comparison-date)))))
 
+;; Valid time HH:MM
+(defmethod validate-rule :time [_ _ data _ _ & [message]]
+  (when
+    (and (not (empty-value? data)) (not (re-matches #"^(0[0-9]|1[0-9]|2[0-3])(:[0-5][0-9])?$" data)))
```

Comment:
We also need to support "." as separator between hour and minute, as is recommended for finnish.